### PR TITLE
Remove descriptions

### PR DESCRIPTION
**Pull request by magdapoppins**
As roi has moved on to use the new details info, yeet the unused descriptions.

Also added tags and descriptions to many images. Please check if I'm being a fool, for example in: 
```
busybox:1.28.3:
  isRecommended: False
  description: 'Tiny versions of many common UNIX utilities'
  tags:
    - '<5Mb'
    - 'UNIX'
  aliases:
    - busybox:1
    - busybox:1.28
```

### CODE REVIEW

**Comment by akx** [comment]
valohai-cli's quickstart also uses this file - maybe schema removals should happen in e.g. a `images.v2.yaml` so we can migrate all tools to it?

**Review by akx — CHANGES_REQUESTED** [review]
valohai-cli's quickstart also uses the images.yaml file - maybe schema removals should happen in e.g. a images.v2.yaml so we can migrate all tools to it?

File: images.v2.yaml [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    - tensorflow/tensorflow:1.13.1-py3
+    - tensorflow/tensorflow:1.15.0-gpu-py3
+    - tensorflow/tensorflow:1.15.2-gpu-py3
+    - tensorflow/tensorflow:2.0.0-gpu-py3
+    - tensorflow/tensorflow:2.0.1-gpu-py3
+    - tensorflow/tensorflow:2.2.0-py3
+    - tensorflow/tensorflow:2.2.0-gpu-py3
+    - valohai/keras:2.0.0-tensorflow1.0.1-python3.6-cuda8.0-cudnn5-devel-ubuntu16.04
+    - valohai/keras:2.0.0-theano0.9.0rc4-python3.6-cuda8.0-cudnn5-devel-ubuntu16.04
+    - valohai/darknet:b61bcf5-cuda8.0-cudnn5-devel-ubuntu16.04
```
Comment (akx) [inline images.v2.yaml]: Can we also get rid of this list in v2? (Aren't these the `isRecommended: True` ones?)

Should the aliases also be moved under each `details` entry? 😇 

How about also getting rid of `cached` in v2? It's out of date anyway.

**Review by akx — CHANGES_REQUESTED** [review]

File: images.v2.yaml (hunk 1 above)
Comment (magdapoppins) [reply images.v2.yaml]: Uh, I'm not sure if recommended === suggested? Suggested means they exist, like, everything in the details list. Then, some of them are especially recommended (noted with a fancy badge).

Comment (akx) [reply images.v2.yaml]: Well, should we maybe add `isSuggested: true` then too? Just that now that we are overhauling these, maybe all that data could be in the single `details` dict?

Comment (magdapoppins) [reply images.v2.yaml]: But yes, aliases can move under details and the complete list of details can be used as "suggestions" with info?

**Review by akx — APPROVED** [review]

**Pull request merged**